crates/evm/Cargo.toml

```diff
@@ -59,6 +59,7 @@ revm = { workspace = true, features = ["optional_block_gas_limit", "optional_eip
 sov-modules-api = { path = "../sovereign-sdk/module-system/sov-modules-api", features = ["macros"] }
 sov-prover-storage-manager = { path = "../sovereign-sdk/full-node/sov-prover-storage-manager", features = ["test-utils"] }
 sov-rollup-interface = { path = "../sovereign-sdk/rollup-interface", features = ["testing"] }
+sov-stf-runner = { path = "../sovereign-sdk/full-node/sov-stf-runner" }
```

Comment:
can you also guard this dependency behind native feature below?

Reply:
if dev dependency no need for future gating, no?

Reply:
I also thought that first, but guest code lock files shouldn't have the dependency, but they still do. Maybe @exeokan disabled his rust-analyzer causing lock files to not update automatically? Try running `cargo check`

Reply:
Yes, apparently rust-analyzer didn't update the lock files, and sov-stf-runner is in fact removed from the lock files. No problem here then.